Sources/Location.swift

```diff
@@ -1,13 +1,21 @@
+/// Used to encapsulate a location request
```

Comment:
Could we add some more in-depth description here on what it is used for?
How and when it fits into `Compass` implementations?